.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: elixir
```

Comment:
This doesn't belong to this pull, please open a different pull request if needed.